### PR TITLE
Develop m

### PR DESCRIPTION
**Pull request by MaxiV95**

### CODE REVIEW

File: src/helpers/interpolar.js
Comment (SebaSatelier) [inline src/helpers/interpolar.js:1]: Line 17, if(b < acceleration && acceleration < y)
variable b doesn't exist and the php function is like: 
(y <acceleration && acceleration < _y) or no???

File: src/helpers/peligro.js
Comment (manduinca) [inline src/helpers/peligro.js:1]: Please, better: hazard.js

File: src/helpers/peligro.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    for (let i = 0.1; i < 1.0; i += 0.05) {
+        period.push(i);
+    }
+
+    for (let i = 1.0; i < 3.1; i += 0.1) {
+        period.push(i);
+    }
+
+    let danger = {};
```
Comment (manduinca) [inline src/helpers/peligro.js]: hazard?

File: src/helpers/ibc.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import probabilidades from './probabilidades'
```
Comment (manduinca) [inline src/helpers/ibc.js]: Better: `probabilities`

File: src/helpers/ibc.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import interpolar from './interpolar'
```
Comment (manduinca) [inline src/helpers/ibc.js]: Better: `interpolation`

File: src/helpers/ibc.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    let Sml = Fv * Sl;
+    let Sms = Fa * Ss;
+
+    let Sdl = (2.0 / 3.0) * Sml;
+    let Sds = (2.0 / 3.0) * Sms;
+
+    let T0 = (Sdl / Sds) * 0.2;
+    let Ts = Sdl / Sds;
+
+    let espectro_ibc = {};
```
Comment (manduinca) [inline src/helpers/ibc.js]: `ibc_spectrum`

**Review by manduinca — DISMISSED** [review]
I leaved some comments, I think we can to improve it, but I approved.

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by SebaSatelier — APPROVED** [review]

**Pull request merged**